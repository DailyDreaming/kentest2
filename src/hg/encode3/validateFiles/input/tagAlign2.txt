chr1	222077840	222077865	ATAAACACTGTAGACTTTCCNAGNN	0	+
