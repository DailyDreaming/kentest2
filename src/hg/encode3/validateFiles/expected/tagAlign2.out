Error [file=input/tagAlign2.txt, line=1]: too many mismatches (found 1/25, maximum is 0) (chr1 222077840 222077865 +)
seq=[ATAAACACTGTAGACTTTCCNAGNN]
gen=[aaaaacactgtagactttcccagta]

Aborting .. found 1 error

